.travis.yml

```diff
@@ -3,6 +3,13 @@ language: python
 # Use the new container-based Travis infrastructure.
 sudo: false
 
+# Make sure we are on Ubuntu 14.04
+dist: trusty
+
+branches:
+  except:
+    - /pyup-.*/
```

Comment:
Why? This will inhibit the pyup-branches from being tested. And as far as I understood, one of the main features of pyup is that there are pull-requests which will be tested. Or did I miss something?